<launch>

    <include file="$(find vrx_tasks)/launch/missioncontrol.launch">
        <arg name="world_name" value="scan_and_dock"/>
    </include>



</launch>
